### DISCUSSION TITLE
How to disable auto-fill suggestions in dialogue box?

### DISCUSSION THREAD

**Discussion by Spring2022abcjk** [open-webui/open-webui#7857] [Q&A]
Using Microsoft IME for typing Chinese characters on Windows, I've encountered frequent inaccuracies in auto-fill predictions. Would it be possible to disable this feature until its performance improves?
I also found a related issue on GitHub: #7533

**Comment by TheMasterFX** [comment]
You can disable this feature in the admin-panel => Interface
![image](https://github.com/user-attachments/assets/99f425cd-ad9b-4cb2-9b5d-7f074c4fc2fe)
